code/controllers/subsystem/ticker.dm

```diff
@@ -555,17 +555,21 @@ SUBSYSTEM_DEF(ticker)
 	to_chat(world, "<br><FONT size=3><B>The ship objectives were:</B></FONT>")
 	var/count = 1
 	var/redtext = 0
+	if(!locate(/datum/objective/ftl/gohome) in get_ship_objectives())
+		redtext = 1
```

Comment:
TRUE

Reply:
Why not suggest the changes then ya nerd. Y'all are the ones that made it `=1` in the first place.

Reply:
Resolved in https://github.com/FTL13/FTL13/pull/3654/commits/f62e48889d881f5d826b72a084f180178ef6b938